### DISCUSSION TITLE
This looks great! A few questions.

### DISCUSSION THREAD

**Discussion by TomSmedley** [AntoineW/luge#18] [Q&A] [answered]
Hey @AntoineW,

This looks great. I've been using GSAP for most of my animations but things like mouse based animations are not out-of the box. I like the idea of using the power of CSS for more performant animations over JavaScript. With the library size being 72kb, I was just curious if it comes with everything out of the box such as Lottie which may never be used in a project?

Also I was wondering if the "Front-end animation plugin for uncreative developers" was the correct messaging, I am a creative developer but I can find use for this.

**Comment by AntoineW** [comment, marked as answer]
Hey 👋
I'm glad you like it!

Yes, the final bundle file contains the core and all the plugins, even the ones that you won't use. I'm thinking about a way to create custom builds to remove the unused plugins and save a few kb.
Regarding lottie, the [lottie-web player](https://github.com/airbnb/lottie-web) is not inside the luge bundle so you have to import it separately.

I see what you mean about the messaging. Initially my will was to create a tool that could help regular developers (not creative ones) to integrate animations without a line of JS. Now it's going beyond that and it can also fit well in a creative developer stack, so I agree that the messaging is a bit reductive 🤔
